### PR TITLE
docs: note that --js-flags only affects the browser process

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
This has come up a couple of times in the last week or two, so I just wanted to clarify the docs.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: docs/api/chrome-command-line-switches.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ $ electron --js-flags="--harmony_proxies --harmony_collections" your-app
 
 See the [Node documentation][node-cli] or run `node --help` in your terminal for a list of available flags. Additionally, run `node --v8-options` to see a list of flags that specifically refer to Node's V8 JavaScript engine.
 
+> **NOTE**: This only affects the JavaScript environment of the browser process. Renderer processes are not affected by this flag.
```
Comment (malept) [inline docs/api/chrome-command-line-switches.md]:
```suggestion
> **NOTE**: This only affects the JavaScript environment of the main process. Renderer processes are not affected by this flag.
```

TIL "browser process" vs "[main process](https://github.com/electron/electron/blob/8ba271efba691357ab1970c1a553b4e019125574/docs/glossary.md#main-process)"

Comment (nornagon) [reply docs/api/chrome-command-line-switches.md]: Ah, I didn't realise that Electron had changed the term. 👍

**Comment by nornagon** [comment]
You're totally right, whoops. Good catch.

**Comment by nornagon** [comment]
([here](https://chromium.googlesource.com/chromium/src/+/72.0.3626.84/content/browser/renderer_host/render_process_host_impl.cc#3044) is where the flag gets propagated from the main process to the renderer.)

**Pull request closed without merging**